pkg/migration/azure.go

```diff
@@ -25,10 +27,14 @@ type azureMigratorFactory struct {
 
 type azureMigrator struct {
 	clusterID string
+
+	crs map[string]runtime.Object
+
 	// Migration configuration, dependencies + intermediate cache for involved
 	// CRs.
 	logger       micrologger.Logger
-	wcCtrlClient client.Client
+	mcCtrlClient ctrl.Client
```

Comment:
How is this client is initialized?

Reply:
Ah, forgot the wiring. It's ought to come from the reconciliation logic.